features/step_definitions/basic_steps.rb

```diff
@@ -0,0 +1,11 @@
+Then("I should see the text {string}") do |text|
+  expect(page).to have_content text
+end
+
+Then("I should see the picture {string}") do |pic_url|
+  expect(page).to have_xpath("//img[@src='#{pic_url}']")
```

Comment:
Have you tried has_css?

Reply:
@magnus-thor tried it not working :(

Reply:
I saw that solution too but didn't think it was any nicer that the xpath, since the picture is not css-related per se.